crates/consensus/src/signed.rs

```diff
@@ -45,32 +41,6 @@ impl<T: SignableTransaction<Sig>, Sig> Signed<T, Sig> {
     pub fn signature_hash(&self) -> B256 {
         self.tx.signature_hash()
     }
-
-    /// Output the signed RLP for the transaction.
-    pub fn encode_signed(&self, out: &mut dyn BufMut) {
-        self.tx.encode_signed(&self.signature, out);
-    }
-
-    /// Produce the RLP encoded signed transaction.
-    pub fn rlp_signed(&self) -> Vec<u8> {
-        let mut buf = vec![];
-        self.encode_signed(&mut buf);
-        buf
-    }
-}
-
-impl<T: SignableTransaction<Sig>, Sig> alloy_rlp::Encodable for Signed<T, Sig> {
```

Comment:
i assume these were replaced by manual impls? if so, we should also note to implementers of `SignedTransaction` that they need to provide these by themselves, since it is needed to convert mempool tx -> typed tx

another q here: is it even possible to impl this in an external crate because of orphan rule? if not, we need to keep this

Reply:
i see we can't blanket impl this anymore